From foo@baz Wed Jun 28 07:56:26 AM CEST 2023
Date: Wed, 28 Jun 2023 07:56:26 +0200
To: Greg KH <gregkh@linuxfoundation.org>
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Subject: act_mirred: remove unneded merge conflict markers

In commit 169a41073993 ("act_mirred: use the backlog for nested calls to mirred
ingress"), a merge conflict marker snuck in, so remove it.

Reported-by: Bart Van Assche <bvanassche@google.com>
Fixes: 169a41073993 ("act_mirred: use the backlog for nested calls to mirred ingress")
Cc: Sasha Levin <sashal@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 tools/testing/selftests/net/forwarding/tc_actions.sh |    1 -
 1 file changed, 1 deletion(-)

--- a/tools/testing/selftests/net/forwarding/tc_actions.sh
+++ b/tools/testing/selftests/net/forwarding/tc_actions.sh
@@ -199,7 +199,6 @@ mirred_egress_to_ingress_tcp_test()
 	log_test "mirred_egress_to_ingress_tcp ($tcflags)"
 }
 
->>>>>>> e921d05033293 (act_mirred: use the backlog for nested calls to mirred ingress)
 setup_prepare()
 {
 	h1=${NETIFS[p1]}
